README.md

```diff
@@ -1,3 +1,5 @@
+>ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.
```

Comment:
Should that be in the [license](https://github.com/signalfx/signalfx-java#license) section?

Reply:
For SignalFx branded repos, we add the notice on top. For Splunk branded ones - the recent ones, we add it at the very end.

Reply:
👍